src/F3DLoader.cxx

```diff
@@ -41,6 +42,12 @@ typedef struct ProgressDataStruct {
 //----------------------------------------------------------------------------
 F3DLoader::F3DLoader()
 {
+  // Disable vtkLogger in case VTK was built with log support
+  vtkLogger::SetStderrVerbosity(vtkLogger::VERBOSITY_OFF);
```

Comment:
not sure what it does.

Reply:
// Disable vtkLogger in case VTK was built with log support 

^^